src/models/subject.ts

```diff
@@ -0,0 +1,50 @@
+import { Classification } from "./decisions/classification";
+import { Verification, VerificationSubject } from "./decisions/verification";
+
+export interface Tag {
+    id?: number;
+    text: string;
+}
+
+export interface ISubjectWrapper {
+    subject: VerificationSubject;
+    identifier: symbol;
+
+    url: string;
+    decisions: [Verification, ...Classification[]] | Classification[];
+    tag: string;
+}
+
+// the Verification model is emitted by the oe-verification-grid as a CustomEvent
+export class SubjectWrapper implements ISubjectWrapper {
+    public constructor(data: ISubjectWrapper) {
+        this.subject = data.subject;
+        this.decisions = data.decisions ?? [];
+        this.url = data.url;
+        this.tag = data.tag;
+
+        this.identifier = Symbol("decision");
+    }
+
+    // aka: context
+    // this is the native data model used by the host application
+    // or this could be the csv row
+    public identifier: symbol;
+    public subject: VerificationSubject;
```

Comment:
why are we double wrapping the subject? A classification has a subject too, but the type name is `VerificationSubject`??